### PR TITLE
PRL-6364

### PR DESCRIPTION
**Pull request by Louis-Solirius**
### Change description

List of pages: 

child-details/add-children?
child-details/ (Provide details for <name>)
child-details/ (Which of the decisions you’re asking the court to resolve relate to <name>)
User(s) to complete event: Citizen

Definition of done:

All POM sections are created
All POMs are added to a relevant journey

### JIRA link

[PRL-6364](https://tools.hmcts.net/jira/browse/PRL-6364)

**Before merging a pull request make sure that:**

- [x] Commits are meaningful and simple
- [x] All commits are squashed into a single commit
- [x] README and other documentation has been updated / added (if needed)

### CODE REVIEW

File: e2e/journeys/citizen/createCase/C100People1/C100People1.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { Page } from "@playwright/test";
```
Comment (JakeCG) [inline e2e/journeys/citizen/createCase/C100People1/C100People1.ts:1]: Can you rename this file just to "People"

File: e2e/pages/citizen/createCase/C100/people1/addChildrenPage.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
+import AccessibilityTestHelper from "../../../../../common/accessibilityTestHelper";
```
Comment (JakeCG) [inline e2e/pages/citizen/createCase/C100/people1/addChildrenPage.ts:1]: Same with this directory

**Review by JakeCG — CHANGES_REQUESTED** [review]

**Review by Scralfie — DISMISSED** [review]

**Review by Scralfie — APPROVED** [review]

**Review by abigailsmith97 — APPROVED** [review]
lgtm

File: e2e/pages/citizen/createCase/C100/people/provideDetailsPage.ts [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  accessibilityTest: boolean;
+  gender: ApplicantGender;
+}
+
+interface fillInFieldsOptions {
+  page: Page;
+  gender: ApplicantGender;
+}
+
+enum radioIds {
```
Comment (JakeCG) [inline e2e/pages/citizen/createCase/C100/people/provideDetailsPage.ts:28]: Missing hashes here

File: e2e/pages/citizen/createCase/C100/people/provideDetailsPage.ts [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+  female = "gender",
+  male = "gender-2",
+  other = "gender-3",
+}
+
+enum inputIds {
```
Comment (JakeCG) [inline e2e/pages/citizen/createCase/C100/people/provideDetailsPage.ts:34]: Missing hashes here

**Review by JakeCG — CHANGES_REQUESTED** [review]

File: e2e/pages/citizen/createCase/C100/people/provideDetailsPage.ts (hunk 3 above)
Comment (Louis-Solirius) [reply e2e/pages/citizen/createCase/C100/people/provideDetailsPage.ts:28]: whoops

**Review by GauravChawlaMOJ — APPROVED** [review]

**Pull request merged**